pages/docs/preload.md

```diff
@@ -114,3 +114,59 @@ db.Preload("Orders.OrderItems.Product").Preload("CreditCard").Find(&users)
 // And GORM won't preload unmatched order's OrderItems then
 db.Preload("Orders", "state = ?", "paid").Preload("Orders.OrderItems").Find(&users)
 ```
+
+## <span id="embedded_preloading">Embedded Preloading</span>
+
+Embedded preloading is used for [Embedded Struct](models.html#embedded_struct), especially the
+same struct. The syntax for Embedded preloading is similar to Nested Preloading, they are divided by dot.
```

Comment:
`Embedded preloading`  `Nested Preloading`
word case

Reply:
Sry, I didn't get it.

Reply:
just keep the case consistent
```diff
- Nested Preloading
+ Nested preloading
```
or
```diff
- Embedded preloading
+ Embedded Preloading
```

Reply:
Do you mean to keep only one of them?
But they're not the same....
I showed this in example
```
// All preloaded include "Address" but exclude "Address.Country", because it won't preload nested associations.
db.Preload(clause.Associations)
```

Reply:
It's not that complicated, just unify words in upper or lower case.

Reply:
Modified. I am too stupid😂😂😂